packages/ui-components/tailwind.config.js

```diff
@@ -7,7 +7,18 @@ export default {
         background: '#242526',
         hero_background: '#22323D',
       },
+      keyframes: {
+        something: {
+          '0%': { opacity: 0, transform: 'translateX(0px)' },
+          '10%': { opacity: 1 },
+          '90%': { opacity: 1 },
+          '100%': { opacity: 0, transform: `translateX(calc(100vw + 1px))` },
+        },
+      },
+      animation: {
+        flow: 'something 1s linear infinite',
+      },
```

Comment:
misschien even de naam van de animation veranderen naar iets duidelijkers 😉.

Reply:
Ah, Jort seems to have already done so in 6b9c4f03d0807e54312bc911e60e2deca92b01d5

Reply:
Sike, seems like Jort is actually the one that named it `something`